### DISCUSSION TITLE
How to invite new users without using Rowy Run?

### DISCUSSION THREAD

**Discussion by yethihahtwe** [AntlerVC/firetable#729] [Support / Q&A] [answered]
Hello.
Is there a way to invite new users without setting up Rowy Run? I cannot upgrade the Firebase plan as banks in my country don't provide credit cards.
I browsed the <projectID>.rowy.app/. Logged in with a new google account. Shows no tables. Then logged out.
User Management from the initial ADMIN account showed "No Roles" for the new user no matter how many times I checked "ADMIN" next to it. It only showed "ADMIN" in Rowy User Management after I added the string "ADMIN" from Firebase Console > _rowy_>userManagement>users>{DocumentID}>roles. But still, I see no tables when logged in with the new account. Please show me how can I manually invite new users.

**Comment by shamsmosowi** [comment, marked as answer]
Hi @yethihahtwe ,
the roles need to be set into the user's auth token custom claims on firebase auth. Managing firebase auth can't be done on the client side, which introduces the need for deploying the cloud run instance.
However you can run firebase admin sdk locally to set the roles without having to enable billing
https://docs.rowy.io/setup/roles?set-user-roles-tabs=admin-sdk#set-user-roles-tabs

**Reply by yethihahtwe** [reply]
Thanks, I'll try out.
